src/ucs/config/global_opts.c

```diff
@@ -416,15 +396,42 @@ static ucs_status_t ucs_vfs_write_log_level(void *obj, const char *buffer,
     return UCS_OK;
 }
 
-/**
- * VFS nodes representing global options are removed in UCS_STATIC_CLEANUP of
- * vfs_obj.c file. Because removing elements using ucs_vfs_obj_remove in
- * UCS_STATIC_CLEANUP in global_opts.c could happen after execution of
- * UCS_STATIC_CLEANUP of vfs_obj.c file.
- */
-UCS_STATIC_INIT
+void ucs_global_opts_init()
 {
+    ucs_status_t status;
+
+    UCS_CONFIG_ADD_TABLE(ucs_global_opts_table,  &ucs_config_global_list);
+    UCS_CONFIG_ADD_TABLE(ucs_global_opts_read_only_table,
+                         &ucs_config_global_list);
+
+    status = ucs_config_parser_fill_opts(&ucs_global_opts,
+                                         ucs_global_opts_read_only_table,
+                                         UCS_DEFAULT_ENV_PREFIX, NULL, 1);
+    if (status != UCS_OK) {
+        ucs_fatal("failed to parse global runtime read-only configuration -"
```

Comment:
minor: pls remove "- aborting" in both places

Reply:
> minor: pls remove "- aborting" in both places

removed